app/scripts/controllers/transactions/index.js

```diff
@@ -958,6 +958,7 @@ export default class TransactionController extends EventEmitter {
           txMeta.txParams.from,
           txMeta.destinationTokenDecimals,
           approvalTxMeta,
+          txMeta.chainId,
```

Comment:
Looking at the number of arguments going to this function, we're going to want to shift to passing an object sooon.

Reply:
I have put this changed on another branch and will PR it once this PR is merged https://github.com/MetaMask/metamask-extension/pull/new/getSwapsTokensReceivedFromTxMeta-object-param